pkg/controller/node/node_controller.go

```diff
@@ -595,15 +540,26 @@ func (r *ReconcileNode) Reconcile(request reconcile.Request) (reconcile.Result,
 		}
 		reqLogger.Info("Updating node tag for port", "port.Id", lsp.Id)
 		if foundNodeTag == false {
-			var nodeTag = common.Tag{Scope: nodeNameScope, Tag: request.Name}
+			var nodeTag = model.Tag{Scope: &nodeNameScope, Tag: &request.Name}
 			lsp.Tags = append(lsp.Tags, nodeTag)
 		}
 		if foundClusterTag == false {
-			var clusterTag = common.Tag{Scope: clusterScope, Tag: cluster}
+			var clusterTag = model.Tag{Scope: &clusterScope, Tag: &cluster}
 			lsp.Tags = append(lsp.Tags, clusterTag)
 		}
-		nsxClient := nsxClients.ManagerClient
-		_, _, err = nsxClient.LogicalSwitchingApi.UpdateLogicalPort(nsxClient.Context, lsp.Id, *lsp)
+		connector := nsxClients.PolicyConnector
+		segPortStateClient := segSDK.NewPortsClient(connector)
+		segId, err := operatortypes.ExtractSegmentIdFromPath(*lsp.ParentPath)
+		if err != nil {
+			cachedNodeSet[nodeName] = &statusmanager.NodeStatus{
+				Addresses: nodeAddresses,
+				Success:   false,
+				Reason:    fmt.Sprintf("Error while achieving port with specific address for node %s: %v", nodeName, err),
```

Comment:
I know this error message has been like this for year, do you mind improving it?
It goes into the node status and not just in the logs.

Maybe just use "retrieving" instead of "achieving" (and do this for all the place where we ser this error in status)

Reply:
done